gm4_metallurgy_moneo/data/moneo_shamir/functions/active.mcfunction

```diff
@@ -1,27 +1,19 @@
 #run from main
 #@s = players holding a moneo tool
 
+tag @s add gm4_has_moneo
+
 scoreboard players set tool_current_damage gm4_ml_data 0
 execute store result score tool_current_damage gm4_ml_data run data get entity @s SelectedItem.tag.Damage
 
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:wooden_shovel"}}] run function moneo_shamir:tools/materials/wood
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:wooden_pickaxe"}}] run function moneo_shamir:tools/materials/wood
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:wooden_hoe"}}] run function moneo_shamir:tools/materials/wood
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:wooden_axe"}}] run function moneo_shamir:tools/materials/wood
-
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:stone_shovel"}}] run function moneo_shamir:tools/materials/stone
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:stone_pickaxe"}}] run function moneo_shamir:tools/materials/stone
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:stone_hoe"}}] run function moneo_shamir:tools/materials/stone
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:stone_axe"}}] run function moneo_shamir:tools/materials/stone
-
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:iron_shovel"}}] run function moneo_shamir:tools/materials/iron
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:iron_pickaxe"}}] run function moneo_shamir:tools/materials/iron
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:iron_hoe"}}] run function moneo_shamir:tools/materials/iron
-execute if entity @s[nbt={SelectedItem:{id:"minecraft:iron_axe"}}] run function moneo_shamir:tools/materials/iron
+execute unless score @s gm4_moneo_max matches 0.. run function moneo_shamir:store_maximum
```

Comment:
Wouldn’t this have the wrong max if you switch moneo tools?

Reply:
Oh right, good point

Reply:
I guess that's what I get for working on it at 2-3am

Reply:
This should now be resolved.